src/main/java/strategy/handlers/ReturnHandler.java

```diff
@@ -0,0 +1,12 @@
+package strategy.handlers;
+
+import db.Storage;
+import model.FruitTransaction;
+import strategy.OperationHandler;

```

Comment:
It's good practice to separate imports by groups and follow the Java convention: first should be standard Java imports, then third-party libraries, and after that your project imports. Make sure there is an empty line between these groups.